package.json

```diff
@@ -51,8 +51,10 @@
     "@gemini-testing/commander": "2.15.3",
     "@types/mocha": "10.0.1",
     "@wdio/globals": "8.21.0",
+    "@wdio/protocols": "8.22.0",
     "@wdio/types": "8.21.0",
-    "@wdio/utils": "7.26.0",
+    "@wdio/utils": "8.35.0",
+    "@wdio/utils-cjs": "npm:@wdio/utils@7.26.0",
```

Comment:
Currently we have `webdriverio: 8.21.0`, but we didn't update `@wdio/utils` because it is not support cjs from `@wdio/utils@8` (only esm). On client side cjs version of `@wdio/utils` didn't build correctly. So use old version for nodejs env (until we move to esm) and new version for browser env.